extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/AmazonLambdaRecorder.java

```diff
@@ -6,12 +6,14 @@
 import java.net.URL;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import com.fasterxml.jackson.databind.MapperFeature;
```

Comment:
```suggestion
```

Reply:
made a mistake with the squash and reverted my commit that contained the `mvn process-sources` by mistake. It's fixed now in one commit.